app/routes/surveys/components/SurveyEditor.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+
+import styles from './surveys.css';
+import React, { Component } from 'react';
+import { DetailNavigation, ListNavigation } from '../utils.js';
+import { Field } from 'redux-form';
+import Button from 'app/components/Button';
+import {
+  TextInput,
+  RadioButton,
+  SelectInput,
+  RadioButtonGroup,
+  DatePicker
+} from 'app/components/Form';
+import { createValidator, required } from 'app/utils/validation';
+import { reduxForm } from 'redux-form';
+import type { SurveyEntity } from 'app/reducers/surveys';
+import Content from 'app/components/Layout/Content';
+
+type Props = {
+  survey: SurveyEntity,
+  submitting: boolean,
+  handleSubmit: ((SurveyEntity) => Promise<*>) => void,
+  autoFocus: any,
+  submitFunction: (SurveyEntity, ?number) => Promise<*>,
+  deleteSurvey: number => Promise<*>,
+  push: string => void
+};
+
+class SurveyEditor extends Component<Props> {
+  onSubmit = (formContent: Object) => {
+    const { survey, submitFunction, push } = this.props;
+    return submitFunction({
+      ...formContent,
+      event: formContent.event && Number(formContent.event.value),
+      surveyId: survey && survey.id
+    }).then(result => {
+      const id = survey ? survey.id : result.payload.result;
+      push(`/surveys/${String(id)}`);
+    });
+  };
+
+  render() {
+    const {
+      survey,
+      submitting,
+      autoFocus,
+      handleSubmit,
+      deleteSurvey
+    } = this.props;
+
+    const titleField = (
+      <Field
+        placeholder="Tittel"
+        label=" "
```

Comment:
nice label

Reply:
Actually intentional, since this is one of those fields that are supposed to look like the page turned editable. Same argument as in the bdb. Since I use `NavigationTab` and make it a Field, using a label looks pretty out of place. Gotta manually include this empty one because it uses some other default otherwise.